package.json

```diff
@@ -1,26 +1,34 @@
 {
   "name": "npm-migrate",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "Migrate all versions of a selected package from a registry to another one",
   "main": "npm-migrate.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/necccc/npm-migrate.git"
+  },
   "keywords": [
     "npm",
     "migration",
     "npme"
   ],
   "author": "nec@shell8.net",
   "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/necccc/npm-migrate/issues"
+  },
+  "homepage": "https://github.com/necccc/npm-migrate#readme",
   "dependencies": {
-    "async": "^2.1.4",
-    "compressing": "^1.1.0",
-    "graceful-fs": "^4.1.11",
-    "lodash.curry": "^4.1.1",
-    "mute": "^2.0.6",
-    "mv": "^2.1.1",
-    "npm": "^4.2.0",
-    "rimraf": "^2.5.4"
+    "async": "2.1.4",
+    "compressing": "1.1.0",
+    "graceful-fs": "4.1.11",
+    "lodash.curry": "4.1.1",
+    "mute": "2.0.6",
+    "mv": "2.1.1",
+    "npm": "4.2.0",
+    "rimraf": "2.5.4"
```

Comment:
I dropped the `^` as there is no `package.lock.json` file. So this was an easy way to lock them down.

Reply:
I can add a `package.lock.json` if you'd like me to